lib/AppInfo/Capabilities.php

```diff
@@ -161,7 +161,7 @@ private function getCapabilitiesCircleConstants(): array {
 				Circle::CFG_BACKEND => $this->l10n->t('Backend'),
 				Circle::CFG_LOCAL => $this->l10n->t('Local'),
 				Circle::CFG_ROOT => $this->l10n->t('Root'),
-				Circle::CFG_CIRCLE_INVITE => $this->l10n->t('Circle Invite'),
+				Circle::CFG_CIRCLE_INVITE => $this->l10n->t('Team Invite'),
```

Comment:
We use sentence case, so better "Team invite".

Reply:
Done